specification/v0/FilecoinDataPreparationManifestSpecification.md

```diff
@@ -10,79 +10,160 @@ Data Preparation Manifest Specification versions follow [SemVer 2.0.0](https://s
 
 ## Super-manifest
 
-[!TODO]
-Separate into 2 sections: a table of properties with longer form descriptions, links etc, and then an example.
+A super-manifest describes a complete dataset which may be stored as several pieces (due to storage segment size constraints) and is a JSON document with the following content:
+
+| Property key    | Data type | Validation  | Description |
+| ----            | ----      | ----        | ----        |
+| @spec           | String    | required, utf-8, URL, max-len 256    | URL for version of the manifest specification used for this manifest. |
+| @spec_version   | String    | required, utf-8, [SemVer 2.0.0](https://semver.org/), max-len 32  | Version number of the specification used for this manifest. |
+| @type           | String    | required, utf-8 "super-manifest"     | This is a super-manifest.  |
+| name            | String    | required, utf-8, max-len 128         | Name of the dataset. |
+| description     | String    | required, utf-8, max-len 4096        | Description of the dataset. |
+| license         | String    | required, utf-8, [SPDX-License-Identifier](https://spdx.org/licenses/), max-len 64  | License(s) the dataset is distributed under. |
+| url             | String    | required, utf-8, [URL](https://datatracker.ietf.org/doc/html/rfc1738), max-len 2048  | Dataset project URL. |
+| uuid            | String    | required, utf-8, [UUID v4](https://datatracker.ietf.org/doc/html/rfc4122)  | Tooling assigned unique ID for this preparation of the dataset. |
+| n_pieces        | Number    | required, positive integer  | Number of pieces making up the complete dataset. |
+| tags            | Array(String) | optional, array max-len 32, string utf-8, max-len 64  | User supplied tags for the dataset. |
+| pieces          | Array(Piece) | required  | Definition of all the pieces that make up the dataset. |
+
+A Piece has the following content:
+
+| Property key    | Data type | Validation  | Description |
+| ----            | ----      | ----        | ----        |
+| piece_cid       | String    | required, utf-8, [CID v1](https://docs.ipfs.tech/concepts/content-addressing/#version-1-v1)  | Content Identifier for the piece (as in the Filecoin deal). |
+| payload_cid     | String    | required, utf-8, [CID v1](https://docs.ipfs.tech/concepts/content-addressing/#version-1-v1)  | Content Identifier for the payload (CAR) in the piece. |
+| contents        | Array(Entry) | optional  | Definition of the files and directories in the piece.  |
```

Comment:
"Definition" again. 

But bigger comment: maybe a link to the definition of `Entry`? It's quite long separated in the doc.

Reply:
Link added, thanks